node/src/main.rs

```diff
@@ -19,12 +26,17 @@ pub struct Cli {
 pub enum Command {
     /// Start the node
     Start {
-        #[arg(short, long, value_name = "FILE", default_value = config::CONFIG_FILENAME)]
+        #[arg(short, long, value_name = "FILE", default_value = NODE_CONFIG_FILE_PATH)]
         config: PathBuf,
     },
 
     /// Generate genesis file
     MakeGenesis {
+        /// Read genesis file inputs from this location
+        #[arg(short, long, value_name = "FILE", default_value = DEFAULT_GENESIS_INPUTS_PATH)]
```

Comment:
Is using `value_name = "FILE"` here for both commands correct? When I execute `miden-node make-genesis --help`, I get the following:
```
Generate genesis file

Usage: miden-node make-genesis [OPTIONS]

Options:
  -i, --input-path <FILE>          Read genesis file inputs from this location [default: genesis.toml]
  -o, --output-path <OUTPUT_PATH>  Write the genesis file to this location [default: genesis.dat]
  -f, --force                      Generate the output file even if a file already exists
  -h, --help                       Print help
```

Reply:
Are you saying that you would prefer to have <CONFIG> and <INPUT_PATH> instead. Or should we put something else?

Reply:
I'm actually not sure what `value_name` is supposed to represent. Having `output-path <OUTPUT_PATH>` or `input-path <INPUT_PATH>` is kind of redundant, I think. Maybe we don't need it at all?

Reply:
Having not build the CLI myself I gave a deeper look into the Clap crate, it is seems that Clap does not provide a standard way to not display value_name for commands that take options.

Hence we could keep it as <FILE> or something else. Or find a way to hide it ( could be time-consuming ). 

I found an example with another instantiation of that seemed to remove the value_name, but that would need for me to make a PR related to changing the CLI structure.

Reply:
Yeah - no need to do anything custom here. What happens if we just remove `value_name` from the attribute? I.e., do something like this:
```Rust
MakeGenesis {
    /// Read genesis file inputs from this location
    #[arg(short, long, default_value = DEFAULT_GENESIS_INPUTS_PATH)]
    inputs_path: PathBuf,

    /// Write the genesis file to this location
    #[arg(short, long, default_value = DEFAULT_GENESIS_FILE_PATH)]
    output_path: PathBuf,

    /// Generate the output file even if a file already exists
    #[arg(short, long)]
    force: bool,
},
```